pkg/apis/infra/v1alpha1/cluster.go

```diff
@@ -86,8 +88,50 @@ const (
 	ScalingDownPhase CustomClusterPhase = "ScalingDown"
 )
 
+const (
+	// VMsClusterReadyCondition reports on whether the VMs cluster is provisioned.
+	VMsClusterReadyCondition capiv1.ConditionType = "VMsClusterReady"
+	// InitInProgressReason (Severity=Warning) documents that the initialization worker is currently running to deploy the VMs cluster with the desired state.
+	InitInProgressReason = "InitInProgress"
```

Comment:
InitInProgress is not a reason, what do you want to represent with it

Reply:
Because I want to present some information about the worker being executed with this conditions.
 I was inspired by this：
 https://github.com/kubernetes-sigs/cluster-api/blob/85c6a451c835998f980f0dbbe62890134c3121ba/controlplane/kubeadm/internal/controllers/controller.go#L381

Reply:
is this equal to `Provisioning` phase?